content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,255 @@
+recorded. Click into the :guilabel:`Project` drop-down menu to select a project.
+
+To create a new project where the timesheets are recorded, click into the :guilabel:`Project`
+drop-down menu, type a name for the project, and then click :guilabel:`Create` from the drop-down
+menu beneath.
+
+.. image:: track_and_bill/track-bill-enable-settings.png
+   :align: center
+   :alt: The settings page of a Helpdesk team with the track and bill time settings enabled.
+
+.. _helpdesk/configure-service-products:
+
+Configure service products
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the :guilabel:`Time Billing` feature is enabled, a new product is created in the *Sales* app
+called *Service on Timesheets*. This product can be found under :menuselection:`Sales app-->
+Products --> Products`. Then, search for `Service on Timesheets` in the :guilabel:`Search...` bar.
+This is the product that is used when invoicing for *post-paid support services* **after** they have
+been completed.
+
+Select :guilabel:`Service on Timesheets` from the product page. This reveals the product detail
+form. The product is configured with the :guilabel:`Product Type` set to :guilabel:`Service` and the
+:guilabel:`Invoicing Policy` set to :guilabel:`Based on Timesheets`. Make any necessary changes to
+the product record, such as the :guilabel:`Cost` or :guilabel:`Sales Price`.
+
+.. image:: track_and_bill/track-bill-product-based-on-timesheets.png
+   :align: center
+   :alt: The Service on Timesheets product with emphasis on the invoicing policy field.
+
+In order to invoice for support services *before* the work has been completed, also known as
+*prepaid support services*, a separate product with a different invoicing policy must be created.
+
+To create a new service product, go to :menuselection:`Sales app --> Products --> Products`, and
+click :guilabel:`Create`. This reveals a blank product detail form.
+
+On the new product form, add a :guilabel:`Product Name`, and set the :guilabel:`Product Type` to
+:guilabel:`Service`. Then, set the :guilabel:`Invoicing Policy` to :guilabel:`Prepaid/Fixed Price`.
+This means an invoice can be generated and payment can be received for this product before any
+timesheets entries have been recorded for these services.
+
+.. image:: track_and_bill/track-bill-product-prepaid-fixed.png
+   :align: center
+   :alt: The Service on Timesheets product with emphasis on the invoicing policy field.
+
+Finally, set the :guilabel:`Sales Price`, and confirm that the :guilabel:`Unit of Measure` is set to
+:guilabel:`Hours`.
+
+Invoice prepaid support services
+================================
+
+When support services are billed on a fixed price, an invoice can be created before any work is
+completed on the issue. In this case, a service product with the *Invoicing Policy* set to
+*Prepaid/Fixed Price* would be used, just like the one created in :ref:`the section above
+<helpdesk/configure-service-products>`.
```

Comment:
is this like doc links, where the name of the sectin shoudl be used instead of a generic 'secton above'? I know for docs I was told to do that, so now I list the name of the doc instead of say "this documentation".